crates/ledger/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "amaru-ledger"
+version.workspace = true
+edition.workspace = true
+description.workspace = true
+license.workspace = true
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+documentation.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+ouroboros = { path = "../ouroboros" }
+async-trait.workspace = true
+gasket.workspace = true
+hex.workspace = true
+num = { version = "0.4.3", features = ["num-bigint"] }
```

Comment:
Any reason why this and some others aren't pulled from the workspace directly?

Reply:
I pulled up the ones used in multiple crates only. Alternatively, I could pull them all. WDYT?